pkg/util/otel/auto-instrumentation.go

```diff
@@ -0,0 +1,23 @@
+package otel
```

Comment:
rename the file to `_` to be consistent with other files.

Reply:
just having `.../otel/auto.go` would drive point home for me, but otherwise I agree with andrii 👍

Reply:
renamed it, `auto` is too generic.

Reply:
And "auto instrumentation" is kind of a known term in the OTel world.